log4j-core-its/src/test/java/org/apache/logging/log4j/core/async/perftest/ResponseTimeTest.java

```diff
@@ -141,7 +141,7 @@ public static void main(final String[] args) throws Exception {
         runLatencyTest(logger, WARMUP_DURATION_MILLIS, WARMUP_COUNT, loadMessagesPerSec, idleStrategy,
                 warmupServiceTmHistograms, warmupResponseTmHistograms, threadCount);
         System.out.println("-----------------Warmup done. load=" + loadMessagesPerSec);
-        if (!Constants.ENABLE_DIRECT_ENCODERS || !Constants.ENABLE_THREADLOCALS) {
+        if (!Constants.ENABLE_DIRECT_ENCODERS || !org.apache.logging.log4j.util.Constants.isUseThreadLocals()) {
```

Comment:
I didn't want to refactor all the use of constants in 2.x as that's something related to the context-scoped properties feature introduced in 3.x. However, if we have tests that would otherwise need to, say, modify a static final field, then yes, I'd refactor this.

Reply:
So we are converting constants that are subject to modification in tests to `Lazy*` instances and leaving out the rest intact?

Reply:
Yes, that was the idea here. In main, that's something we're trying to clean up more for the properties enhancement, but it requires a lot of refactoring, hence why it's limited to main.